lib/react_client.dart

```diff
@@ -860,7 +861,6 @@ class ReactDomComponentFactoryProxy extends ReactComponentFactoryProxy {
 
   /// Prepares the bound values, event handlers, and style props for consumption by ReactJS DOM components.
```

Comment:
#supernit this comment could be updated since "bound" values don't exist anymore... But that can certainly be done in one of the follow-up PRs.

Reply:
Updated